File: lec01/03_sequentialSearch/start/sequentialSearch.py
# sequentialSearch.py
# TODO: 배열에서 특정 값을 순차적으로 탐색하는 함수를 작성하세요.
def sequential_search(arr, target):
  """순차 탐색 함수"""
  # 여기에 코드를 작성하세요
  return -1


# 사용 예시
if __name__ == "__main__":
  arr = [64, 34, 25, 12, 22, 11, 90]
  target = 22

  result = sequential_search(arr, target)

  if result != -1:
    print(f"값 {target}를 인덱스 {result}에서 찾았습니다.")
  else:
    print(f"값 {target}를 찾을 수 없습니다.")

File: lec01/03_sequentialSearch/start/sequentialSearch.c
// sequentialSearch.c
#include <stdio.h>

// TODO: 배열에서 특정 값을 순차적으로 탐색하는 함수를 작성하세요.
// 순차 탐색 함수
int sequentialSearch(int arr[], int size, int target) {
  // 여기에 코드를 작성하세요
  return -1;
}

// 사용 예시
int main() {
  int arr[] = {64, 34, 25, 12, 22, 11, 90};
  int size = sizeof(arr) / sizeof(arr[0]);
  int target = 22;

  int result = sequentialSearch(arr, size, target);

  if (result != -1) {
    printf("값 %d를 인덱스 %d에서 찾았습니다.\n", target, result);
  } else {
    printf("값 %d를 찾을 수 없습니다.\n", target);
  }

  return 0;
}
